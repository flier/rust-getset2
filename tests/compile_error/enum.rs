use getset2::{Getter, Setter};

#[derive(Getter)]
pub enum Enum {
    Foo,
    Bar,
}

#[derive(Setter)]
pub enum Enum2 {
    Foo,
    Bar,
}

fn main() {}
