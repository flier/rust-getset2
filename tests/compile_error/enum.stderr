error: #[derive(Getter)] can only be applied to structure
 --> tests/compile_error/enum.rs:4:1
  |
4 | / pub enum Enum {
5 | |     Foo,
6 | |     Bar,
7 | | }
  | |_^

error: #[derive(Setter)] can only be applied to structure
  --> tests/compile_error/enum.rs:10:1
   |
10 | / pub enum Enum2 {
11 | |     Foo,
12 | |     Bar,
13 | | }
   | |_^
